controllers/admin/AdminAddressesController.php

```diff
@@ -129,7 +129,7 @@ public function renderForm()
                     'name' => 'dni',
                     'required' => false,
                     'col' => '4',
-                    'hint' => $this->trans('DNI / NIF / NIE', array(), 'Admin.OrdersCustomers.Feature')
+                    'hint' => $this->trans('The national ID card number of this person, or a unique tax identification number.', array(), 'Admin.OrdersCustomers.Feature')
```

Comment:
Yup. Anywhere, really. So that any kind of number can be entered in that field / DB row.